src/index.html

```diff
@@ -4,9 +4,464 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="content">
+    <header class="header">
+      <div class="logo header__logo">
+        <a 
+          class="logo__link" 
+          href="#"
+        >
+          <img 
+            class="logo__img" 
+            src="./images/logo.png" 
+            alt="MOYO Logo"
+          >
+        </a>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements

![2023-02-08_15h22_56](https://user-images.githubusercontent.com/61904995/217542139-d41116da-77d1-4599-8f41-77d9f6161815.png)